### ISSUE TITLE
Is the API good enough?

### ISSUE THREAD

**Issue by slorber** [slorber/rereduce#1]

**Comment by yelouafi** [comment]
considering the example in README

``` javascript
const secondReducer = createReducer({ firstReducer },
  function (state = 0,action,{ firstReducer }) {
    return firstReducer + 1
  }
)
```

`secondReducer` doesn't react to any action, neither it needs any state; all it does is deriving some data from another; It appears more a selector than a reducer

**Comment by slorber** [comment]
yes maybe it's not the best example but I don't have any concrete usecase I'd like to solve :) (see https://github.com/rackt/redux/issues/1315#issuecomment-177946101)
